fvm/state/state_test.go

```diff
@@ -92,36 +92,6 @@ func TestState_ChildMergeFunctionality(t *testing.T) {
 
 }
 
-func TestState_InteractionMeasuring(t *testing.T) {
```

Comment:
the coverage of this removed test case has already been taken care of in https://github.com/onflow/flow-go/blob/master/fvm/meter/weighted_meter_test.go#L380